tests/custom/test_compression_disabled.py

```diff
@@ -47,91 +49,91 @@ def test_compression_disabled(tpuf: Turbopuffer):
         assert perf.client_deserialize_ms > 0
 
 
-def test_compression_disabled_constructor():
-    # Test that compression can be disabled via constructor
-    client = Turbopuffer(
-        api_key="test",
-        region="aws-us-east-2", 
-        compression=False
-    )
-    assert client.compression is False
-
-
```

Comment:
@rkruze this is the key part of the change; I moved the `assert ... is False` into the main test case above.